### PR TITLE
Book detail iv

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary
- The html content of the whole book-detail was separated and added to the [detail.html ](https://github.com/wolox-training/jlb-angular/blob/book-detail-IV/detail.html) file
- Badge image was positioned relative to the cover component

## Screenshot
book-detail-IV

## Trello card
https://trello.com/c/X7dlsqNx/29-detalle-de-libro-iv-badge

### CODE REVIEW

**Review by CrisVega08 — CHANGES_REQUESTED** [review]
Same

File: detail.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="container detail">
+    <div class="card">
+      <div class="cover">
+        <img class="cover__image" src="assets/book-cover.png" alt="Book cover">
+        <div class="cover__badge">
+          <img class="cover__badge__image" src="assets/badge.png" alt="Cover badge">
```
Comment (eledwinn) [inline detail.html]: Avoid to use HTML elements for decorative elements. You can use ::after or ::before pseudoelements.

File: scss/detail.scss [hunk 2]
```diff
@@ -1,74 +1,84 @@
...
-  &__image {
-    height: inherit;
-    width: auto;
+    &__image {
+      height: inherit;
+      width: auto;
+    }
+
+    &__badge {
+      height: 118.57px;
```
Comment (eledwinn) [inline scss/detail.scss]: 🤔

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 3]
```diff
@@ -1,74 +1,84 @@
...
+      position: absolute;
+      right: 0;
+      top: 0;
+      transform: translate(30%, -15%) rotate(15deg);
+      width: 94.53px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
      width: 94px;
```

File: scss/detail.scss (hunk 2 above)
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
      height: 118px;
```

File: scss/detail.scss [hunk 4]
```diff
@@ -1,74 +1,84 @@
-.container {
+.detail {
```
Comment (CrisVega08) [inline scss/detail.scss:1]: Something very important you don't need move sass classes when add new parent element, this a specificity to selector and we don't want it

so use classes in this form 

.detail {
  ...
}

.card {
  ...
}

card don't need be inside .detail

File: scss/detail.scss [hunk 5]
```diff
@@ -1,74 +1,84 @@
...
+    width: 100%;
+  }
 
-.cover {
-  display: flex;
-  flex-direction: column;
-  height: 368px;
-  margin-right: 52px;
-  overflow: hidden;
+  .cover {
```
Comment (CrisVega08) [inline scss/detail.scss]: same

File: scss/detail.scss [hunk 6]
```diff
@@ -1,74 +1,84 @@
...
+    }
   }
-}
 
-.info {
-  color: $black;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
+  .info {
```
Comment (CrisVega08) [inline scss/detail.scss]: Same

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/styles.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.container {
```
Comment (CrisVega08) [inline scss/styles.scss:1]: Where do you use this class ? and why in other file?

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: detail.html (hunk 1 above)
Comment (Juliocho990) [reply detail.html]: Great!

File: scss/detail.scss (hunk 2 above)
Comment (Juliocho990) [reply scss/detail.scss]: I'm going to approach it at 120px

Comment (Juliocho990) [reply scss/detail.scss]: Ok, I used 120px to be multiplo of 4px

File: scss/detail.scss (hunk 4 above)
Comment (Juliocho990) [reply scss/detail.scss:1]: Thanks Cristian I will keep this in mind

File: scss/detail.scss (hunk 5 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok

File: scss/detail.scss (hunk 6 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok

File: scss/styles.scss (hunk 7 above)
Comment (Juliocho990) [reply scss/styles.scss:1]: I have used that class to wrap the content of the each view, it is probable that I will make changes regarding the way to implement it.

**Review by eledwinn — APPROVED** [review]

File: scss/detail.scss [hunk 8]
```diff
@@ -22,12 +20,25 @@
   display: flex;
   flex-direction: column;
   height: 368px;
-  margin-right: 52px;
-  overflow: hidden;
+  margin: 0 52px 0 0;
+  position: relative;
```
Comment (CrisVega08) [inline scss/detail.scss:24]: I dont understand why no use margin-right: 52px?;

File: scss/detail.scss [hunk 9]
```diff
@@ -69,4 +80,4 @@
   &__item--secundary {
     color: $gray;
   }
-}
+}
```
Comment (CrisVega08) [inline scss/detail.scss]: Add empty line

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 3 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok!

File: scss/detail.scss (hunk 8 above)
Comment (Juliocho990) [reply scss/detail.scss:24]: I already changed it, thanks

File: scss/detail.scss (hunk 9 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok!

**Review by CrisVega08 — APPROVED** [review]

**Review by eledwinn — APPROVED** [review]

**Pull request merged**